### DISCUSSION TITLE
Sparse CTRL release

### DISCUSSION THREAD

**Discussion by DavideAlidosi** [continue-revolution/sd-webui-animatediff#406] [Q&A] [answered]
Hi, judging from the branches on Git and at the various repositories it would seem that support for Sparse CTRL has ended. Why not release a version that includes this critical component and add the new features to a later version?
Thanks

**Comment by continue-revolution** [comment, marked as answer]
That’s because I want to release both sparsectrl and magic animate together. I met some difficulties figuring out how to do magic animate, which block me from any other further updates. Please give more patience.

Fortunately I had some idea yesterday and can hopefully carry out soon.

**Reply by DavideAlidosi** [reply]
Thank's for reply!

**Reply by continue-revolution** [reply]
Alright it’s released now.
